docs/data-ai/capture-data/capture-other-sources.md

```diff
@@ -0,0 +1,11 @@
+---
+linkTitle: "Capture other data sources"
+title: "Capture and sync other data sources"
+tags: ["data management", "data", "services"]
+weight: 12
+layout: "docs"
+type: "docs"
+platformarea: ["data"]
+description: "TODO"
```

Comment:
still a draft. if anybody has any better ideas for a name for this besides "capture other sources" I'm all ears

Reply:
sgtm